calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+    <td><p style="text-align: center;">Foreground extraction</p></td>
+  </tr>
+</table>
+
+### Step 3: Matching and filtering
+
+Since it is not possible to directly differentiate reflector detections, whether there are multiple targets in the calibration area or if the detections are from humans or radar reflectors, we rely on the initial calibration to pair each lidar detection with its closest radar detection, and vice versa. A detection pair is accepted if they are mutually their the closest matches. Once a match is made, it is evaluated against existing hypotheses (monitored by a Kalman filter): if it aligns with an existing hypothesis, that hypothesis is updated; if it does not align with any, a new hypothesis is created. When a hypothesis achieves convergence, it is finalized and added to the calibration list.
+
+### Step 4: Calibration
+
+After matching detection pairs from the sensors, we can compute the transformation between them using estimation algorithms. Currently, we support two methods: a 2d SVD-based approach and a yaw-only rotation approach.
+
+For the 2d SVD-based method, since radar detections lack a Z component, we convert the problem to 2d by setting the Z component of lidar detections to zero. We then estimate the transformation using the SVD method provided by PCL.
+
+The yaw-only rotation method, on the other hand, calculates the average yaw angle difference of all pairs and estimates the transformation, considering only rotation, between the sensors. Generally, the 2d calibration is preferred when valid; otherwise, the yaw rotation is used as the calibration output.
+
+It's also important to note that in the near future, the calibrator will be updated to support radar that includes elevation angle and provides different transformation algorithms.
+
+### Diagram
+
+Below, you can see how the algorithm is implemented in the `marker_radar_lidar_calibrator` package.
+
+![marker_radar_lidar_calibrator](../docs/images/marker_radar_lidar_calibrator/marker_radar_lidar_calibrator.jpg)
+
+## ROS Interfaces
+
+### Input
+
+| Name                     | Type                            | Description                                |
+| ------------------------ | ------------------------------- | ------------------------------------------ |
+| `input_lidar_pointcloud` | `sensor_msgs::msg::PointCloud2` | Topic of lidar pointcloud for calibration. |
+| `input_radar_msg`        | `radar_msgs::msg::RadarTracks`  | Topic of radar objects for calibration.    |
+
+### Output
+
+| Name                          | Type                                   | Description                                               |
+| ----------------------------- | -------------------------------------- | --------------------------------------------------------- |
+| `lidar_background_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Publishes the background pointcloud from lidar.           |
```

Comment:
You are describing a noun, yet start the description with a verb

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/67263aa5b911e572a1d0d9cf1d6607b5bebd1df9